locales/en/metrics.json

```diff
@@ -53,5 +53,12 @@
   "total_bytes": "Bytes incoming and outgoing",
   "total_bytes_popover_header": "Bytes incoming and outgoing",
   "used_disk_space": "Used disk space",
-  "used_disk_space_help_text": "Used disk space is the amount of disk space used by the Kafka broker in the instance. This metric enables you to assess available disk space relative to the limit. To reduce used disk space, you can adjust topic retention time or other topic properties as needed."
+  "used_disk_space_help_text": "Used disk space is the amount of disk space used by the Kafka broker in the instance. This metric enables you to assess available disk space relative to the limit. To reduce used disk space, you can adjust topic retention time or other topic properties as needed.",
+  "partition_limit": "Limit 1000 partitions.",
```

Comment:
As @riccardo-forina pointed out, hard-coded values should be pulled from the API. This limit of `1000` can vary depending on instance type and size.

Reply:
One other update here...

Can you remove the period "." at the end of this line? It shouldn't have been included in the original storybook implementation, since this isn't a complete sentence. 

@riccardo-forina Should a PR be opened in the ui-components repo to make this update there, too?

Reply:
Oh, this is the UI components repo. Sorry, it has been a long week.